packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -7,16 +7,56 @@ import classNames from 'classnames';
 import { useFocusOnTriggeringElementOnClose } from '../../lib/hooks';
 import { Link, RouteComponentProps, useLocation } from '@reach/router';
 
+type ModalButtonProps = {
+  ctaText: string;
+  className?: string;
+  revealModal: () => void;
+  modalRevealed?: boolean;
+  alertBarRevealed?: boolean;
+};
+
+export const ModalButton = ({
+  ctaText,
+  className,
+  revealModal,
+  modalRevealed,
+  alertBarRevealed,
+}: ModalButtonProps) => {
+  const modalTriggerElement = useRef<HTMLButtonElement>(null);
+  // If the UnitRow children contains an AlertBar that is revealed,
+  // don't redirect focus back to the element that opened the modal
+  // because focus will be set in the AlertBar.
+  useFocusOnTriggeringElementOnClose(
+    modalRevealed,
+    modalTriggerElement,
+    alertBarRevealed
+  );
+
+  return (
+    <button
+      className={classNames('cta-base transition-standard ml-2', className)}
```

Comment:
I think this is missing a `cta-neutral` class:

<img width="545" alt="Screen Shot 2020-09-08 at 3 06 07 PM" src="https://user-images.githubusercontent.com/6392049/92517599-e4b31300-f1e4-11ea-8d3d-7dd0044a3269.png">